src/main/java/uk/gov/hmcts/reform/cwrdapi/config/RepositoryConfig.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.cwrdapi.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.support.SimpleJpaRepository;
+import uk.gov.hmcts.reform.cwrdapi.domain.CaseWorkerAudit;
+import uk.gov.hmcts.reform.cwrdapi.domain.ExceptionCaseWorker;
+
+import javax.persistence.EntityManager;
+import javax.transaction.Transactional;
+
+@Configuration
+public class RepositoryConfig {
+
+    @Autowired
+    EntityManager entityManager;
+
+    @Bean
```

Comment:
do you really need this class? why cant we use Repositories classes directly ?

Reply:
Yes for blank method it redundant to use create repositories and use simple repository. That is it's whole purpose.